db/migrate/20190702061900_create_users.rb

```diff
@@ -1,7 +1,6 @@
 class CreateUsers < ActiveRecord::Migration[5.2]
   def change
     create_table :users do |t|
-      t.string :email
```

Comment:
Sao lai sua migrate kieu nay.
Lam the nay thi moi nguoi khong ai chay dc rails db:migrate het

Reply:
Yep. để e viết lại migrate ạ